cli/src/funTest/assets/semver4j-analyzer-result.yml

```diff
@@ -222,7 +222,7 @@ scanner:
               name: "ScanCode"
               version: "3.2.1-rc2"
               configuration: "--copyright --license --ignore *.ort.yml --info --strip-root\
-            \ --timeout 300 --ignore HERE_NOTICE --ignore META-INF/DEPENDENCIES --json-pp"
+            \ --timeout 300 --ignore META-INF/DEPENDENCIES --json-pp"
```

Comment:
Commit message nit: "..**is** simply..."

Reply:
Done.